R/init.R

```diff
@@ -138,7 +140,7 @@ init <- function(data,
 
   # resolve modules datanames
   datanames <- teal.data::get_dataname(data)
-  join_keys <- data$get_join_keys()
+  join_keys <- teal.data::get_join_keys(data)
```

Comment:
I believe this line makes the `datanames` argument to `new_teal_data` mandatory (docs say "required when...").